.github/workflows/e2e-matrix.yml

```diff
@@ -214,6 +217,8 @@ jobs:
             project: e2e-angular-core
           - node_version: 16
             project: e2e-angular-extensions
+          - node_version: 16
+            project: e2e-angular-module-federation
```

Comment:
It should also be added for `node_version: 20`

Reply:
it is? right? 
<img width="642" alt="image" src="https://github.com/nrwl/nx/assets/12140467/a6710f09-c219-4cb2-a0e7-8811e478a351">

Reply:
That's correctly added to the matrix, but we exclude the other angular e2e projects from running with node 16 and 20. The new one need to match that.